apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -385,7 +402,57 @@ export default class Utilities {
     console.log(os.EOL + 'Running "npm install" in ' + directory);
 
     // NOTE: Here we use whatever version of NPM we happen to find in the PATH
-    Utilities.executeCommandWithRetry('npm', ['install'], maxInstallAttempts, directory, suppressOutput);
+    Utilities.executeCommandWithRetry(maxInstallAttempts, 'npm', ['install'], directory,
+      Utilities._createEnvironmentForRushCommand(''),
+      suppressOutput);
+  }
+
+  /**
+   * Returns a process.env environment suitable for executing lifecycle scripts.
+   * @param initCwd - The INIT_CWD environment variable
+   * @param initialEnvironment - an existing environment to copy instead of process.env
+   */
+  private static _createEnvironmentForRushCommand(initCwd: string,
```

Comment:
>initCwd: string [](start = 50, length = 15)

options object?

Reply:
Since this is private, we can do this when we add more parameters.

---
In reply to: [184192083](https://github.com/Microsoft/web-build-tools/pull/623#discussion_r184192083) [](ancestors = 184192083)